src/qml/LocationMarker.qml

```diff
@@ -57,9 +57,14 @@ Item {
         }
     }
 
+    property int lastAcceptedReading: 0
     onReadingChanged: {
-      compassOrientation = userOrientation + (-(Math.atan2(reading.x, reading.y) / Math.PI) * 180)
-      compassHasValue = true
+      var timestamp = Date.now();
+      if (timestamp - lastAcceptedReading > 500) {
```

Comment:
```suggestion
      if (timestamp - lastAcceptedReading > 5000) {
```

Reply:
Isn't this 5 seconds instead of 0.5 secs?

Reply:
@suricactus , that was just me doing a test, back to 500.

Reply:
Stupid me, sorry for the noise.